io.c

```diff
@@ -9710,7 +9714,8 @@ static VALUE argf_readlines(int, VALUE *, VALUE);
  *    readlines(sep, limit, **line_opts) -> array
  *
  *  Returns an array containing the lines returned by calling
- *  Kernel#gets until the end-of-file is reached.
+ *  Kernel#gets until the end-of-file is reached;
+ *  (see {Lines}[#class-IO-label-Lines]).
```

Comment:
This method (`File#readlines`) is in `File.html` so I think we need to link to `IO.html` for the link to work.

```suggestion
 *  (see {Lines}[IO.html#class-IO-label-Lines]).
```

Reply:
Fixed.  Fwiw, this is the global readlines, in Kernel.  Still needed your fix.